docs/Deploying-PowerAuth-Standard-RESTful-API.md

```diff
@@ -0,0 +1,52 @@
+# Deploying PowerAuth Standard RESTful API Bootstrap
+
+This chapter explains how to deploy PowerAuth Standard RESTful API.
+
+PowerAuth Standard RESTful API is a Java EE application (packaged as an executable WAR file) responsible for exposing the [RESTful API according to the specification](https://github.com/wultra/powerauth-crypto/docs/Standard-RESTful-API.md). It exposes services for end-user applications (PowerAuth Clients), such as the mobile banking app or mobile token app.
```

Comment:
`https://github.com/wultra/powerauth-crypto/docs/...` should be `https://github.com/wultra/powerauth-crypto/blob/develop/docs/...`